### PR TITLE
resource_control: add repeats to `tidb_runaway_queries`

### PR DESCRIPTION
**Pull request by HuSharp**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/55664

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilin90 — APPROVED** [review]
/lgtm

File: tidb-resource-control.md [hunk 1]
```diff
@@ -330,19 +330,24 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
-    其中，`match_type` 为该 Runaway Query 的来源，其值如下：
+    字段解释：
 
-    - `identify` 表示命中条件。
-    - `watch` 表示被快速识别机制命中。
+    - `match_type` 为该 Runaway Query 的来源，其值如下：
+        - `identify` 表示命中条件。
+        - `watch` 表示被快速识别机制命中。
+    - `start_time` 为该 Runaway Query 被识别的时间。
+    - `repeats` 为该 Runaway Query 从 `start_time` 开始后被识别的次数。
```
Comment (hfxsd) [inline tidb-resource-control.md:350]:
```suggestion
    - `start_time` 为该 Runaway Query 被识别的时间。
    - `repeats` 为该 Runaway Query 从 `start_time` 开始后被识别的次数。
    - `match_type` 为该 Runaway Query 的来源，其值如下：
        - `identify` 表示命中条件。
        - `watch` 表示被快速识别机制命中。
```

**Review by hfxsd — APPROVED** [review]

**Review by glorv — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Comment by lilin90** [comment]
Resolved a conflict via https://github.com/pingcap/docs-cn/pull/18611/commits/6f55f2de87d09d636569bb5e62e9671f786ed648.

**Pull request merged**